packages/store/src/internals.ts

```diff
@@ -67,9 +88,9 @@ export function fastPropGetter(paths: string[]): (x: any) => any {
  *  };
  *
  */
-export function buildGraph(states) {
-  const findName = klass => {
-    const meta = states.find(g => g === klass);
+export function buildGraph(stateKlasses: StateKlass[]): StateKeyGraph {
+  const findName = (klass: StateKlass): string => {
+    const meta: StateKlass = stateKlasses.find(g => g === klass);
```

Comment:
Won't it imply this?

Reply:
Yes, I will move to implied where possible